src/azure-cli/HISTORY.rst

```diff
@@ -25,6 +25,8 @@ Release History
 **Compute**
 
 * vm create: Add warning when specifying accelerated networking and an existing NIC together.
+* vm create: Add --vmss to specify virtual machine scale set that the virtual machine should be assigned to.
+* vmss create: Add --empty to create a VMSS without VM profile, which can be then referenced in `vm create`.
```

Comment:
Empty keyword will give a wrong indication IMO cause one can create an empty VMSS even when VM profile exists. We have started to use orchestrationMode to specify this. The 2 modes are: VMSS and VM. I'd also seek feedback from Jun and Varun.

Reply:
Also vmss create should happen before vm create in this case. Not sure ordering matters in the doc.

Reply:
> 
> 
> Also vmss create should happen before vm create in this case. Not sure ordering matters in the doc.

This doesn't matter. It's just change log.

Reply:
> 
> 
> Empty keyword will give a wrong indication IMO cause one can create an empty VMSS even when VM profile exists. We have started to use orchestrationMode to specify this. The 2 modes are: VMSS and VM. I'd also seek feedback from Jun and Varun.

I think "empty" is ambiguous, too. Do you have a suggested name?
Is there a change of vm and vmss creation ways in the future? You mentioned orchestrationMode.